src/platforms/apple/common/dsym.mdx

```diff
@@ -34,40 +36,50 @@ Setting up App Store Connect currently requires two separate credentials.
 
 Follow the documentation on [Creating API Keys for App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and provide the _Issuer_, _Key ID_, and _Private Key_. The API Key needs to have the _Developer_ role in order to fetch build details.
 
-**iTunes Credentials**:
+**App Store Connect Credentials**:
```

Comment:
Hey @philipphofmann, I'm not sure the renaming of "iTunes" to "App Store Connect" in this section is great, these are not App Store Connect specific credentials after all and naming them "iTunes credentials" makes it very clear what kind of credentials are expected here.  Also, the UI shows these as "iTunes Credentials" when prompting for them during the setup.  What was the motivation behind doing this?  Would changing this back be bad?

Reply:
I think you already changed it. Thanks @flub.